src/BridgedRoom.ts

```diff
@@ -660,7 +660,11 @@ export class BridgedRoom {
             return;
         }
 
-        const reactionKey = emoji.emojify(`:${message.reaction}:`, getFallbackForMissingEmoji);
+        let reactionKey = emoji.emojify(`:${message.reaction}:`, getFallbackForMissingEmoji);
+        // Element uses the default thumbsup and thumbsdown reactions with an appended variant character.
+        if (reactionKey === '👍' || message.reaction === '👎') {
+            reactionKey += '\ufe0f'.normalize(); // VARIATION SELECTOR-16
```

Comment:
it's only the :+1: and :-1: emoji Element does this to?!

Reply:
I tested the other quick reactions and they work fine. (2 means that Matrix and Slack are sending the same character)
![Screenshot_2020-09-24_21-56-39](https://user-images.githubusercontent.com/10872136/94194503-12b97800-feb2-11ea-8c9b-51fdd5a3ae63.png)

Reply:
From the MSC: 

> When sending emoji reactions, the key field should include the colourful variation-16 when applicable.

why is it only applicable to append VS16 to :+1:  and :-1: ?!

Reply:
It's not applicable to everything, but I'm 97% sure there's more than :+1:/:-1: